distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -497,9 +497,6 @@ private static Integer findPortNumber(Integer portToTry) {
                         RMI_SERVER_PORT.getPort()),
                 installStartupFile(getClass().getClassLoader()
                         .getResourceAsStream("hazelcast.xml"), "/etc/hazelcast.xml"),
-                installStartupFile(getClass().getClassLoader()
```

Comment:
❓ I want to clarify that removing these lines will have the below side effects in two different scenarios. Let me know if this is correct or not.

**The distribution does NOT have a `ddf.security.sts.client.configuration.config` file in `etc` by default.**
 - Pax exam puts an empty one in `etc`.
 - The lines below this will modify the empty version

**The distribution does have a `ddf.security.sts.client.configuration.config` file in `etc` by default.**
 - The lines below this modify that config file, but only change the `address` property.